### DISCUSSION TITLE
A red screen of death. Why and how to fix it?

### DISCUSSION THREAD

**Discussion by tom-avilius** [hyprwm/Hyprland#6743] [Q&A]
It used to happen ever so rarely, but now it happens almost every time I leave my machine to get water, tea, etc. In general, whenever I leave it for a while.
I have researched the issue and met dead ends, some issues are still open while others have been closed with a commit that addresses it. All my packages however are updated and I still face the issue. 

It is a light red screen. Sometimes the cursor is visible but mostly not. Can't do nothing other than physically switching my PC off. I am using Hyprlock.

**Comment by MightyPlaza** [comment]
swaylock/hyprlock crashed

**Reply by tom-avilius** [reply]
Is there anything I can do other than disabling the lock altogether?

**Reply by MightyPlaza** [reply]
you probably want to find out why it is crashing and report the issue on the hyprlock repo
